src/types/registration.types.ts

```diff
@@ -83,6 +83,8 @@ interface AdditionalIdentifier {
   value: string;
 }
 
+type RegistrationOperation = 'update' | 'delete' | 'unpublish';
```

Comment:
Slette avpublisert resultat
```suggestion
type RegistrationOperation = 'update' | 'delete' | 'unpublish' | 'terminate';
```